src/App.tsx

```diff
@@ -1,12 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './app.scss';
+//import { Repo } from './components/Repo';
+import  WelcomePage from './components/WelcomePage'
 
 export const App: React.FC = () => {
```

Comment:
```suggestion
export const App = () => {
```

`React.FC` is discouraged 
[link](https://github.com/facebook/create-react-app/pull/8177)